src/Notifications/NotificationCenterController.cs

```diff
@@ -71,6 +78,11 @@ private void WebView_CoreWebView2InitializationCompleted(object sender, Microsof
 
             if (notificationUIPopup.webView.CoreWebView2 != null)
             {
+                // More initialization options
+                // Context menu disabled
+                notificationUIPopup.webView.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
+                // Opening hyper-links using default system browser instead of WebView2 tab window
+                notificationUIPopup.webView.CoreWebView2.NewWindowRequested += WebView_NewWindowRequested;
```

Comment:
For DocumentationBrowser I used a different approach for opening links(using the NavigationStartingEvent) in the Default Web Browser, if you want to take a quick look here is the code (a similar code was also used for opening links in MS WebBrowser).
https://github.com/DynamoDS/Dynamo/blob/538dfbeaaa3bdfcdd570417b7d513373473e3d7b/src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs#L87